packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -75,6 +88,114 @@ ModifierButton.propTypes = {
   }).isRequired,
 };
 
+const isBlockActive = (editor, type, level = undefined) => {
+  const { selection } = editor;
+
+  if (!selection) return false;
+
+  let matchCondition;
+
+  switch (type) {
+    case 'paragraph':
+      matchCondition = (n) => n.type === type;
+      break;
+    case 'heading':
+      matchCondition = (n) => n.type === type && n.level === level;
+      break;
+    default:
+      matchCondition = (n) => n.type === type;
+      break;
+  }
+
+  const match = Array.from(
+    Editor.nodes(editor, {
+      at: Editor.unhangRange(editor, selection),
+      match: (n) => !Editor.isEditor(n) && SlateElement.isElement(n) && matchCondition(n),
+    })
+  );
+
+  return match.length > 0;
+};
+
+const toggleBlock = (editor, format, blockType = 'paragraph', level = undefined) => {
+  Transforms.unwrapNodes(editor, {
+    match: (n) => !Editor.isEditor(n) && SlateElement.isElement(n),
+    split: true,
+  });
+
+  let newProperties = {
+    type: blockType,
+    level: level || undefined,
+  };
```

Comment:
Here when it's not a heading level is undefined, but the entry should not exist at all

```js
// what we want
{
  type: 'list',
}

// what we don't want
{
  type: 'list',
  level: undefined,
}
```

also it looks like it could be a const or just an inline value